manage_tool_dependencies.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
It may be useful to have a short comment to explain what this script is suppose to do. It seems to me it is starting the tool migrations, so the file name may be adjusted too.

Also, not sure we want to clutter the main directory with rarely used scripts, maybe move it to `scripts/` ?